packages/@lwc/integration-karma/test/light-dom/scoped-slot/if-block/index.spec.js

```diff
@@ -2,19 +2,25 @@ import { createElement } from 'lwc';
 
 import MixedSlotParent from 'x/mixedSlotParent';
 
+const vFragBookend = process.env.API_VERSION > 59 ? '<!---->' : '';
+
 describe('if-block', () => {
     it('should work when parent and child have matching slot types', () => {
         const elm = createElement('x-parent', { is: MixedSlotParent });
         elm.showStandard = true;
         document.body.appendChild(elm);
         const child = elm.shadowRoot.querySelector('x-mixed-slot-child');
-        expect(child.innerHTML).toBe('<span>Slotted content from parent</span>');
+        expect(child.innerHTML).toBe(
```

Comment:
We have been conveniently using `.innerHTML` to test the rendered output. However, it appears those tests have to do with leaky abstractions from the LWC engine now that we are using comments. 

It doesn't have to be done in this PR, but I would suggest moving away from `.innerHTML` when not needed to avoid running into this issue. This observation is applicable to all the tests below.

Reply:
I'll open a work item to capture this work. I agree, these tests are more fragile than I'd prefer.

Reply:
Created W-14449676.